tests/integration/external/drupal6/skipif.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+if (version_compare(PHP_VERSION, "7.4", ">")) {
+  die("skip: PHP > 7.4.0 not supported\n");
```

Comment:
Is only 7.4.0 supported and not 7.4.x where x != 0?

Reply:
This skips PHP versions 7.4.x and above. 7.3.12 will pass but 7.4.12 will not.